app/models/dossier.rb

```diff
@@ -857,6 +861,7 @@ def after_passer_en_construction
       .passer_en_construction
       .processed_at
     save!
+    self.update_procedure_dossiers_count
```

Comment:
```suggestion
    procedure.compute_dossiers_count
```

J'ai l'impression que ce n’est pas nécessaire d'exposer la méthode intermédiaire sur le dossier

Reply:
Effectivement, ce n'est plus nécessaire. J'ai modifié en conséquence ici : https://github.com/demarches-simplifiees/demarches-simplifiees.fr/pull/8369/commits/c3dd0d15f9aea7421edaf2fc547a633dba3467ad